source/accessible-media/production-captions.html.erb.md

```diff
@@ -89,35 +94,58 @@ formatted and timestamped.
 
 ### Caption quality
 
-Always provide the highest-quality captions that convey 100% accuracy.
+{::nomarkup}
+<%= ref :start %>
+{:/nomarkup}
+
 See [resources for writing quality captions](#resources-for-writing-quality-captions) for more
-information. Keep the following basic points in mind when writing
+information.
+
+{::nomarkup}
+<%= ref :middle %>
+{:/nomarkup}
+
+Always provide the highest-quality captions that convey 100% accuracy.
+Keep the following points in mind when writing
 captions:
 
 -   Ensure that there are no spelling errors. This includes the names of
     characters or speakers.
 -   Use conventional grammar rules. After end punctuation (period,
-    question mark, exclamation point, etc.), always begin a new caption.
+    question mark, exclamation point, etc.), always begin a new caption block.
```

Comment:
although the end-punctuation rule i mention is a general rule, your example is correct.  but that's probably too much detail for this tutorial; it's more appropriate for a caption-style manual.